bme280/bme280.go

```diff
@@ -52,9 +69,34 @@ func New(bus drivers.I2C) Device {
 	}
 }
 
-// Configure sets up the device for communication and
-// read the calibration coefficientes.
+// ConfigureWithSettings sets up the device for communication and
+// read the calibration coefficients.
+//
+// The default configuration is the Indoor Navigation settings
+// from the BME280 datasheet.
 func (d *Device) Configure() {
+	d.ConfigureWithSettings(Config{})
+}
+
+// ConfigureWithSettings sets up the device for communication and
+// read the calibration coefficients.
+//
+// The default configuration if config is left at defaults is
+// the Indoor Navigation settings from the BME280 datasheet.
+func (d *Device) ConfigureWithSettings(config Config) {
```

Comment:
Why have both Configure & ConfigureWithSettings ? Shouldn't it be better to just call Configure(config Config) ?

Reply:
I had it that way and @deadprogram asked to separate out to avoid breaking back-compat since existing code had 'Configure' with no parameters.

Reply:
Right, just see the reply, sorry for the question.